index.html

```diff
@@ -667,6 +667,20 @@ <h1>Trust Model</h1>
 <a>relying party</a>, a more flexible and dynamic trust model is created
 such that market competition and customer choice is increased.
       </p>
+
+      <p class="note">
+Experts may recognize that the data model detailed in this specification
```

Comment:
Should avoid qualifying readers as "experts" or not.  Perhaps remove that first part and make it a statement like "The data model detailed in this specification..."